doc/index.md

```diff
@@ -0,0 +1,338 @@
+
+![Search Using Search Bar](./images/searchusingsearchbar.png)
+
+```csharp
+if (geocodeResult != null)
+{
+    // create a picture marker symbol and offset it
+    var uiImagePin = UIImage.FromBundle("MapPin");
+    var mapPin = this.ImageToByteArray(uiImagePin);
+    var roomMarker = new PictureMarkerSymbol(new RuntimeImage(mapPin));
+    roomMarker.OffsetY = uiImagePin.Size.Height * 0.65;
+
+    // Create graphic
+    var mapPinGraphic = new Graphic(geocodeResult.DisplayLocation, roomMarker);
+
+    // Add pin to map
+    var graphicsOverlay = this.MapView.GraphicsOverlays["PinsGraphicsOverlay"];
+    graphicsOverlay.Graphics.Clear();
+    graphicsOverlay.Graphics.Add(mapPinGraphic);
+    this.MapView.GraphicsOverlays["PinsGraphicsOverlay"].IsVisible = true;
+}
+```
+
+### Querying data
+
+The locator provides a map location for the searched feature, but does not provide any attribute data. To populate the card at the bottom of the map and select the appropriate floor, the data is queried
+
+```csharp
+// Run query to get attributes of the selected room
+var roomsLayer = this.Map.OperationalLayers[AppSettings.CurrentSettings.RoomsLayerIndex] as FeatureLayer;
+
+if (roomsLayer != null)
+{
+    // Get feature table to be queried
+    var roomsTable = roomsLayer.FeatureTable;
+
+    // Set query parameters
+    var queryParams = new QueryParameters()
+    {
+        ReturnGeometry = true,
+        WhereClause = string.Format(string.Join(" = '{0}' OR ", AppSettings.CurrentSettings.LocatorFields) + " = '{0}'", searchString)
+    };
+
+    // Query the feature table
+    var queryResult = await roomsTable.QueryFeaturesAsync(queryParams);
+        return queryResult.FirstOrDefault();
+    }
+}
+```
+
+### Setting a home location
+
+The app offers the option to set a Home Location. This location is used to pre-populate the starting point of routes, and the user can easily get back Home by tapping the Home button to the right of the search bar. The same locator is used to generate the suggestions and retrieve the selected location. Once set, the Home location is written to the AppSettings file and persists between app sessions.
+
+|Settings with home|Go to home|
+|---	|---	|
+|![Settings with home](./images/settingswithhome.png)|![Go to home](./images/gotohome.png)|
+
+### Routing
+
+Once search is complete and a feature is selected on the map, the route icon appears on the bottom card. Tapping that navigates the app to the routing page, where user is prompted to add a Start and End location. The End location is always pre-populated with the feature selected on the map. The Start location is only pre-populated if Location Services is enabled or if the user has a Home Location set. When the user hits Route Me, the app navigates back to the map and the route is generated using the TransportationNetwork that was packaged in the mmpk. [Read more about routing and how to create routes using transportation networks](/net/latest/ios/guide/find-a-route.htm)
```

Comment:
`[Read more about routing and how to create routes using transportation networks](/net/latest/ios/guide/find-a-route.htm)`

Relative path needs the https://developers.arcgis.com before it